controllers/requests.ts

```diff
@@ -113,9 +114,47 @@ export const getRequestsController = async (req: any, res: any) => {
     if (!requests) {
       return res.status(204).send();
     }
-    return res.status(200).send({
+
```

Comment:
logic for constructing pagination links appears to be duplicated within the `getRequestsController` function. This can lead to maintenance challenges and make the code harder to understand.
 
  I'm suggesting refactoring the logic for constructing pagination links into a reusable function. This will help eliminate duplication and make the codebase easier to maintain.